etc/prod/docker-compose.windows.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.7'
```

Comment:
Maybe this a great opportunity for bump docker-compose version to `3.8`
https://docs.docker.com/compose/compose-file/compose-versioning/#version-38

Reply:
Will require people to upgrade locally and ci for no benefits right now